src/core/renderer.js

```diff
@@ -292,7 +304,7 @@ class Renderer {
 
         const s = 1. / this._zoom;
```

Comment:
Since you have called it `scale` [here](https://github.com/CartoDB/carto-vl/pull/620/files#diff-ba2b399628ae0c09a9b06d41faff9b64R154) you can also refactor this one too :)